plugins/node/instrumentation-undici/package.json

```diff
@@ -51,7 +51,7 @@
     "nyc": "15.1.0",
     "rimraf": "5.0.5",
     "semver": "^7.6.0",
-    "superagent": "8.0.9",
+    "superagent": "9.0.2",
```

Comment:
I don't see this dependency being used anywhere.

@david-luna is it safe to remove it? or am I missing anything?

Reply:
that must be a leftover from the initial commit where I used a `package.json` from another instrumentation as a template. We can close this PR and I'm going to create a PR to remove it ASAP

Reply:
Thanks for taking a look, opened #2239 as a replacement